### PR TITLE
CCITCARBON-207 Modify generic templates for carbon notification to include provision details

### PR DESCRIPTION
**Pull request by shay6**

### CODE REVIEW

File: teflo_webhooks_notification_plugin/templates/slack_template.jinja [hunk 1]
```diff
@@ -38,7 +38,23 @@
                 ]
             },
         {% endif %}
-
+        {% if scenario.get_all_assets() %}
```
Comment (rujutashinde) [inline teflo_webhooks_notification_plugin/templates/slack_template.jinja]: It does not have an else part. if scenario.get_all_assets() is not present

File: teflo_webhooks_notification_plugin/templates/slack_template.jinja [hunk 2]
```diff
@@ -38,7 +38,23 @@
...
+                        "type": "mrkdwn",
+                        "text": """*Assets Section* \n>
+                                    *Asset Name*: {{ asset.name }}\n>
+                                    *IP_Address*: {{ asset.ip_address }} {% if asset.ip_address is defined %} {% else %} No ip_address set {% endif %}\n>
+                                    *Hostname*: {{ asset.hostname }} {% if asset.hostname is defined %} {% else %} No hostname set {% endif %}"""
+                    }
+                },
+            {% endfor %}
+        {% endif %}
         {% if failed_tasks %}
```
Comment (rujutashinde) [inline teflo_webhooks_notification_plugin/templates/slack_template.jinja]: if failed_tasks should come after if passed_tasks

File: teflo_webhooks_notification_plugin/templates/slack_template.jinja (hunk 1 above)
Comment (shay6) [reply teflo_webhooks_notification_plugin/templates/slack_template.jinja]: done

File: teflo_webhooks_notification_plugin/templates/slack_template.jinja (hunk 2 above)
Comment (shay6) [reply teflo_webhooks_notification_plugin/templates/slack_template.jinja]: done

**Review by rujutashinde — COMMENTED** [review]
LGTM

**Review by rujutashinde — APPROVED** [review]

**Pull request merged**